pipeline.py

```diff
@@ -51,19 +58,25 @@ class CorpusManager:
     """
 
     def __init__(self, path_to_raw_txt_data: str):
-        pass
+        self.path_to_raw_txt_data = path_to_raw_txt_data
+        self._storage = {}
+        self._scan_dataset()
 
     def _scan_dataset(self):
         """
         Register each dataset entry
         """
-        pass
+        path = Path(self.path_to_raw_txt_data)
+
+        for file in path.rglob('*_raw.txt'):
```

Comment:
why have you decided to use rglob instead of glob?

Reply:
it wasn't a great idea. changed